### PR TITLE
Test against Rails 5.2

### PR DESCRIPTION
**Pull request by joneslee85**

### CODE REVIEW

File: spec/support/gemfiles/Gemfile.rails-5.2.x [hunk 1]
```diff
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
+
+gemspec :path => '../../../'
+
+gem 'rails', '~> 5.2.0'
+
+# Rails imposed mysql2 version contraints
+# https://github.com/rails/rails/blob/5-2-stable/activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb#L4
```
Comment (orien) [inline spec/support/gemfiles/Gemfile.rails-5.2.x]: Line 6 seems to the appropriate line. (`#6`) Could you please change the comment?

File: .travis.yml [hunk 2]
```diff
@@ -28,3 +28,12 @@ matrix:
...
       env: DB=postgres
+    - rvm: 2.5
+      gemfile: spec/support/gemfiles/Gemfile.rails-5.2.x
+      env: DB=mysql
+    - rvm: 2.5
+      gemfile: spec/support/gemfiles/Gemfile.rails-5.2.x
+      env: DB=sqlite
+    - rvm: 2.5
+      gemfile: spec/support/gemfiles/Gemfile.rails-5.2.x
+      env: DB=postgres
```
Comment (orien) [inline .travis.yml]: 👏 

To reduce build times, could we remove these two from the matrix:

```yaml
     - rvm: 2.5
       gemfile: spec/support/gemfiles/Gemfile.rails-5.1.x
       env: DB=sqlite

     - rvm: 2.5
       gemfile: spec/support/gemfiles/Gemfile.rails-5.1.x
       env: DB=postgres
```

Comment (joneslee85) [reply .travis.yml]: @orien I would love to leave it there to cover these 2 adapters.

File: spec/support/gemfiles/Gemfile.rails-5.2.x (hunk 1 above)
Comment (joneslee85) [reply spec/support/gemfiles/Gemfile.rails-5.2.x]: @orien sorry I do not quite get what you mean

File: .travis.yml (hunk 2 above)
Comment (orien) [reply .travis.yml]: They'll still be covered by the Rails 5.2 entries you've just added.

File: spec/support/gemfiles/Gemfile.rails-5.2.x (hunk 1 above)
Comment (orien) [reply spec/support/gemfiles/Gemfile.rails-5.2.x]: If you follow this link it highlights the wrong line:
https://github.com/rails/rails/blob/5-2-stable/activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb#L4

**Comment by orien** [comment]
We should add Rails 5.2 to the change log too. https://github.com/envato/double_entry/blob/master/CHANGELOG.md

**Pull request merged**